modules/bitgo/example/js/eth/list-wallet-addresses.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * List deployed receive addresses of a eth wallet and their given token balances.
+ *
+ * This tool will help you see how to use the BitGo API to easily list your
```

Comment:
I think this statement will change as this example is related to addresses and not wallets

Reply:
Done.